plugins/module_utils/network/sonic/config/aaa/aaa.py

```diff
@@ -419,85 +391,85 @@ def get_delete_request(self, url, attr):
         request = {'path': url, 'method': DELETE}
         return request
 
-    def get_diff_aaa(self, want, have):
+    def get_diff_aaa(self, base_cfg, compare_cfg):
         """AAA module requires custom diff method due to overwritting of list in SONiC"""
```

Comment:
```suggestion
        """AAA module requires custom diff method due to overwriting of list in SONiC"""
```
Small spelling error correction